### ISSUE TITLE
Failed to install EMS on test environment

### ISSUE THREAD

**Issue by robert-sanfeliu** [eu-nebulous/sal-scripts#13]
On **test** environment installation of EMS fails:

```
[2256t3@ip-172-31-38-34.ec2.internal;m10533-1-master-10533-1_Task_start_0;09:03:02] Error: INSTALLATION FAILED: failed to fetch https://jmarchel7bulls.github.io/helm-charts/ems-server-0.1.0.tgz : 404 Not Found

```

**Comment by robert-sanfeliu** [comment]
The error comes from:
https://github.com/eu-nebulous/monitoring/blob/c608066d2b2172541a557613a31e2a7ab9ba4040/nebulous/ems-nebulous/src/main/resources/helm/epm-deploy.yml#L39

**Comment by robert-sanfeliu** [comment]
Issue already exists on nebulous-cd environment

**Comment by yoctozepto** [comment]
Hmm... Any Helm stuff should be only ever handled from sal-scripts (this repo). The actual installation of EMS server should be happening in:

https://github.com/eu-nebulous/sal-scripts/blob/4f0586ad27fc3aec89decf985316e90acee04dec/installation-scripts-onm/MASTER_START_SCRIPT.sh#L36

```
  sudo -H -E -u ubuntu bash -c 'helm install ems nebulous/ems-server \
  --set tolerations[0].key="node-role.kubernetes.io/control-plane" \
  --set tolerations[0].operator="Exists" \
  --set tolerations[0].effect="NoSchedule" \
  --set app_uuid=$APPLICATION_ID \
  --set broker_address=$BROKER_IP \
  --set image.tag="2024-apr-nebulous" \
  --set broker_port=$BROKER_PORT'
```

**Comment by yoctozepto** [comment]
Based on the on-slack discussion with @ipatini, the script you mention, @robert-sanfeliu, is not invoked. Thus, the issues lies elsewhere. Most likely, SAL has not picked up the updated scripts. If they were updated in-place and SAL was not restarted, then it still uses the old ones. You could verify this by browsing the files SAL currently sees and/or comparing the timestamps on the configmap vs pod.

**Comment by yoctozepto** [comment]
@robert-sanfeliu, @ipatini has pointed out that the failure was on the chart itself and not the repo. I have not backtracked far enough after discovering your analysis was a red herring. ;-) Thus, the following commit should fix the issue:
https://github.com/eu-nebulous/helm-charts/commit/e17ec5620b42bf3e144abdcf96e32270a18e81ca

Please let me know if it is fixed and act on this issue report appropriately.

**Comment by robert-sanfeliu** [comment]
@yoctozepto my bad, I searched for this jmarchel7bulls on the repo and it only showed up one entry for the whole NebulOuS organisation.
![image](https://github.com/eu-nebulous/sal-scripts/assets/172278014/01b1bcd8-3b3f-435e-83f2-6d469c91c00e)

**Comment by yoctozepto** [comment]
> @yoctozepto my bad, I searched for this jmarchel7bulls on the repo and it only showed up one entry for the whole NebulOuS organisation.
> ![image](https://github.com/eu-nebulous/sal-scripts/assets/172278014/01b1bcd8-3b3f-435e-83f2-6d469c91c00e)
> 
> 
> 
> 
> 

That should not have been the case because that subdomain was clearly there before my fix.

Anyhow, please confirm if this helped.

**Comment by yoctozepto** [comment]
@robert-sanfeliu Thanks for confirming. The only difference is that the -test env uses non-ONM scripts.

**Comment by robert-sanfeliu** [comment]
On July 5th I requested @jmarchel7bulls to deploy the latest version of everything (except CFSB) on nebulous-test and use ONM version of the SAL scripts. I think he did, but we can ask him when he is back. Wiki was not updated because I couldn't confirm that the environment was working (and that is why I opened this bug).

**Comment by yoctozepto** [comment]
@robert-sanfeliu You are right, the code is ONM-enabled but both k8s metadata and wiki claim otherwise. I will look into cleaning it up.
